src/main/java/com/napier/sem/DAO.java

```diff
@@ -208,4 +207,27 @@ public static String getWhereCondition(String areaFilter, String areaName)
 
         return executeStatement(statementString, App.CAPITAL_CITY);
     }
+
+    /**
+     * Use cases 6.1-6.3
+     * Constructs an SQL query to fetch the top N populated cities in a specific area, and executes the query.
+     *
+     * @return An ordered list of cities in a defined area sorted by descending population
+     */
+    public ArrayList<Record> topNCapitalCitiesIn(String areaFilter, String areaName, Integer n) {
```

Comment:
Opening bracket on new line!!

Reply:
intellij reformats on it's own but I'll try